src/routes.js

```diff
@@ -15,7 +16,15 @@ const routes = {
   '/admin/reports/dblog': Dblog,
   '/node/add': AddContent,
   '/node/add/recipe': () => (
-    <NodeForm entityTypeId="node" bundle="recipe" widgets={widgets} />
+    <NodeAddForm entityTypeId="node" bundle="recipe" widgets={widgets} />
+  ),
+  '/node/:bundle/:id/edit': ({ match }) => (
```

Comment:
Note: We need to pass along the bundle as there is not way to load a single piece of content without knowing the bundle.